binary_search/C/binary_search.c

```diff
@@ -0,0 +1,27 @@
+#include <stdio.h>
+
+int binarySearch(int arr[], int start, int end, int item)
+{
```

Comment:
{ should almost always be at the end of the previous line  [whitespace/braces] [4]

*Origin: CPPLintBear, Section: `all.cpplint`.*